src/Server.php

```diff
@@ -160,10 +167,21 @@ private function single(Input $input): ?string
      */
     private function validate(stdClass $schema, $data):bool
```

Comment:
At this point I think we should rewrite this method to simply return a `ValidationResult` instead of a bool. It will either be the ValidationResult from the custom Validator or the default one, it won't matter).

Then in the `Server::single()` method we check `isValid()` and do the error formatting when necessary. This should avoid having to introduce this new "error" attrribute.

Reply:
I removed the internal validator and do thome minor fixes. It would be usefull to have a relaes with this changes. At the moment i use my fork for development, but for further steps we internaly block usage of git repos call dev-*